package.json

```diff
@@ -27,7 +37,19 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
```

Comment:
I know you like tooling @rijkvanzanten but isn't all of this a bit too much? Linters seem okay but all the babel and browserify seems a bit over the top?